bin/cml-runner.js

```diff
@@ -327,6 +352,12 @@ const opts = yargs
   .default('name', RUNNER_NAME)
   .describe('name', 'Name displayed in the repository once registered')
 
+  .boolean('retry')
+  .default('retry', RUNNER_RETRY)
+  .describe(
+    'retry',
+    'Automatically retries a run which jobs did not finish due to runner disposal or reached 72 hours in Github'
```

Comment:
```suggestion
    'Automatically retry jobs terminated due to runner disposal or timeout (72 hours on Github)'
```

also should the timeout be configurable (user can set e.g. `--retry-timeout 48h`?)

Reply:
> also should the timeout be configurable (user can set e.g. --retry-timeout 3h for non-self-hosted?)

If is not self hosted there is nothing we can do 😁 they are not running this code